packages/lib/models/utils/onFolderDrop.ts

```diff
@@ -6,7 +6,15 @@ import Folder from '../Folder';
 import Note from '../Note';
 
 export default async (noteIds: string[], folderIds: string[], targetFolderId: string) => {
-	const targetFolder = await Folder.load(targetFolderId, { fields: ['id', 'deleted_time'] });
+	let targetFolder = await Folder.load(targetFolderId, { fields: ['id', 'deleted_time'] });
+	if (targetFolderId === '') {
+		// When dropping to root folder, targetFolderId === ''
+		targetFolder = {
+			id: '',
+			deleted_time: 0,
+			type_: 2,
+		};
+	}
```

Comment:
Sorry I'm not sure that's the right way here, it feels a bit like a hack. Or maybe it is but I'd need to check properly and maybe implement it myself since I recently refactored all this code.

Reply:
This is fine as a solution so feel free to continue if you're interested in completing the pull request. In particular would you be able to add tests for this particular file?